src/components/Community/data.json

```diff
@@ -1,8 +1,8 @@
 {
   "hero": {
-    "pictureDesktop": "/img/community/banner.png",
-    "pictureMobile": "/img/community/banner-mobile.png",
-    "url": "https://www.mlprague.com/#schedule-saturday"
+    "pictureDesktop": "/img/community/header_mailing_list_skinny.png",
+    "pictureMobile": "/img/community/mobile_header_mailing_list_skinny.png",
+    "url": "#subscribe"
```

Comment:
@rogermparent I'm also getting some strange behavior on the mobile version - it does scroll (w/o smooth effect for some reason), but does not open the right point on the page (some text is hidden) ... it might be related or looks similar to #712

Reply:
I have to check this out more closely, thanks for the catch!